pandas/core/frame.py

```diff
@@ -1598,8 +1597,9 @@ def to_feather(self, fname):
         from pandas.io.feather_format import to_feather
         to_feather(self, fname)
 
-    @Substitution(header='Write out column names. If a list of string is given, \
-it is assumed to be aliases for the column names')
+    @Substitution(header='Write out the column names. If a list of strings '
+                         'is given, it is assumed to be aliases for the '
+                         'column names')
```

Comment:
Just wanted to re-add this since my response got hidden: if there are other places that use this documentation, feel free to patch those too.  For bonus points, go ahead and see if you can create a shared resource in a `_shared_docs` dictionary for example that we can import and plugin this docstring.